en/config/safety.md

```diff
@@ -0,0 +1,227 @@
+Return behaviour |  | Choice list of *Return then*: Land, Loiter and do not land, or Loiter and land after a specified time.
+Loiter Altitude | [RTL_DESCEND_ALT](../advanced_config/parameter_reference.md#RTL_DESCEND_ALT) | If return with loiter is selected you can also specify the altitude at which the vehicle hold.
+Loiter Time | [RTL_LAND_DELAY](../advanced_config/parameter_reference.md#RTL_LAND_DELAY) | If return with loiter then land is selected you can also specify how long the vehicle will hold.
+
+> **Note** The return behavour is defined by [RTL_LAND_DELAY](../advanced_config/parameter_reference.md#RTL_LAND_DELAY). If negative the vehicle will land immediately. Additional information can be found in [Return mode](../flight_modes/return.md).
+
+
+### Land Mode Settings
+
+*Land at the current position* is a common [failsafe action](#failsafe_actions) that engages [Land Mode](../flight_modes/land.md). 
+This section shows how to set whether the vehicle will automatically disarm after landing. For Multicopters (only) you can additionally set the descent rate.
+
+![Safety - Land Mode Settings (QGC)](../../images/qgc/setup/safety_land_mode.png)
+
+The settings and underlying parameters are shown below:
+
+Setting | Parameter | Description
+--- | --- | ---
+Disarm After | [COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND) | Select checkbox to specify that the vehicle will disarm after landing, and enter delay after landing before disarming (must be non-zero). 
+Landing Descent Rate | [MPC_LAND_SPEED](../advanced_config/parameter_reference.md#MPC_LAND_SPEED) | Rate of descent (MC only).
+
+
+## Other Failsafe Settings {#failsafe_other}
+
+This section contains information about failsafe settings that cannot be configured through the *QGroundControl* [Safety Setup](#qgc_safety_setup) page.
+
+### Position (GPS) Loss Failsafe
+
+The *Position Loss Failsafe* is triggered if the quality of the PX4 position estimate falls below acceptable levels (this might be caused by GPS loss).
+The action is always to loiter (hover or circle - depending on vehicle type) for a time while trying to regain GPS, and then enter flight termination.
+
+The relevant parameters for all vehicles shown below (also see [GPS Failure navigation parameters](../advanced_config/parameter_reference.md#gps-failure-navigation)):
+
+Parameter | Description
+--- | ---
+[COM_POS_FS_DELAY](../advanced_config/parameter_reference.md#COM_POS_FS_DELAY) | Delay after loss of position before the failsafe is triggered.
+[CBRK_GPSFAIL](../advanced_config/parameter_reference.md#CBRK_GPSFAIL) | Circuit breaker that can be used to disable GPS failure detection.
+[NAV_GPSF_LT](../advanced_config/parameter_reference.md#NAV_GPSF_LT) | Loiter time (waiting for GPS recovery before it goes into flight termination). Set to 0 to disable.
+
+Parameters that only affect Fixed Wing vehicles:
+
+Parameter | Description
```

Comment:
Is this correct? I assume these can only apply to FW

Reply:
Even NAV_GPSF_LT does not apply to multicopters according to @RomanBapst and I checked this in simulation. @bkueng can you please confirm this as well?

Reply:
Good point. But , when does it apply - immediately after failsafe engaged (before doing the failsafe action) or at some other point?